ports/winit/app.rs

```diff
@@ -114,7 +202,24 @@ impl App {
             }
 
             // Handle the event
-            app.queue_embedder_events_for_winit_event(e);
+            let response = match e {
+                winit::event::Event::WindowEvent { ref event, .. } => {
+                    if let Some(mut minibrowser) = app.minibrowser() {
+                        minibrowser.context.on_event(&event)
```

Comment:
There are a few event surges that will return `repaint: true` and make the response of the browser pretty laggy.
These include `WindowEvent::CursorMoved` `WindowEvent::MouseWheel` and `WindowEvent::AxisMotion`.
Not sure what's the proper way to avoid, but I think one workaround could be detecting the height since this is the only egui part it has right now.

Reply:
thanks! i noticed that too, but i wasn’t sure how to fix it yet ^^;;

Reply:
fixed in 62ec187c2d8f89957f64c10a2cd1320ef47ef145 by using winit redraw requests to coalesce minibrowser updates.

ffc971de935aa90979213db8adc8472251e82ecc also tried filtering out CursorMoved events below the toolbar, but this made it impossible to unfocus the location bar by clicking the page contents, so i abandoned that.